### DISCUSSION TITLE
Multi line comments

### DISCUSSION THREAD

**Discussion by jghauser** [nvim-neorg/neorg#412] [Q&A] [answered]
What's the right way of making mult-line comments? Do the following work:
```
@comment
this 
is a

comment
@end

#comment
this is
also a comment
```
I'm asking because there is no highlighting to indicate that these are comments (that is, they're not greyed out like `+inline comments+`. Is it only because the highlighting isn't implemented (yet) or am I getting something wrong? Thanks!

**Comment by vhyrro** [comment, marked as answer]
They're not greyed out because we haven't been able to get our highlight queries to properly work with those for whatever reason (treesitter just bugs out lol).

But yeah, both of those are valid ways of creating comments. `@comment`, `#comment` and `+comment+` are all accepted. Soon this will change with 0.0.11 being released, but our document upgrader in `core.export` will handle that stuff for you, so no need to worry in advance. 
Hope that answers your question!

**Reply by jghauser** [reply]
Thanks!
